debian/control

```diff
@@ -232,6 +232,7 @@ Description: debugging symbols for ceph-mon
 Package: ceph-osd
 Architecture: linux-any
 Depends: ceph-base (= ${binary:Version}),
+         ceph-trace (= ${binary:Version}),
```

Comment:
@liu-chunmei i know you were testing listing the package in 'Recommends' field. Did that not work?